internal/config/config.go

```diff
@@ -27,8 +27,10 @@ type App struct {
 }
 
 type Auth struct {
-	Token          string
-	TokenHeaderKey string
+	Token               string
+	TokenHeaderKey      string
+	ValidationURL       string
```

Comment:
both the auth setup are separate 
one is for gatewayserver other for router

track its config separately, either under `Auth.router`

or `router.Auth`

Reply:
OK, will change it to `auth.router`